src/test/kotlin/no/skatteetaten/aurora/boober/feature/S3FeatureTest.kt

```diff
@@ -339,11 +370,14 @@ class S3FeatureTest : AbstractFeatureTest() {
                 ?: throw Exception("No dc")
 
             val container = dc.spec.template.spec.containers.first()
+            val secretName = "$appName-$bucketObjectArea-s3"
+            val bucketObjectAreaUpper = bucketObjectArea.toUpperCase()
             val actualEnvs = container.env
                 .map { envVar -> envVar.name to envVar.valueFrom.secretKeyRef.let { "${it.name}/${it.key}" } }
+                .filter { (name, _) ->
+                    name.startsWith("S3_BUCKETS_$bucketObjectAreaUpper")
+                }
 
-            val secretName = "$appName-$bucketObjectArea-s3"
-            val bucketObjectAreaUpper = bucketObjectArea.toUpperCase()
             val expectedEnvs = listOf(
                 "S3_BUCKETS_${bucketObjectAreaUpper}_SERVICEENDPOINT" to "$secretName/serviceEndpoint",
```

Comment:
IMHO envvars should start with S3_DEFAULT and S3_NOT_DEFAULT HERE. and not bucketname/objectarea normalized.